src/plugins/GUITestBase/src/runnables/ugene/plugins_3rdparty/primer3/Primer3DialogFiller.cpp

```diff
@@ -128,7 +128,11 @@ void Primer3DialogFiller::commonScenario() {
     }
 
     if (settings.notRun) {
-        GTKeyboardDriver::keyClick(Qt::Key_Escape);
+        if (isOsMac()) {
+            dialog->close();
```

Comment:
You should not call API (main thread) methods in GUI tests. Click something like  'Cancel' or 'OK' button instead

Reply:
That's a good point, fixed